src/energyplus/Test/ReverseTranslator_GTest.cpp

```diff
@@ -548,6 +551,34 @@ TEST_F(EnergyPlusFixture, ReverseTranslator_FoundationKivaSettings) {
   EXPECT_EQ("Hourly", foundationKivaSettings.simulationTimestep());
 }
 
+TEST_F(EnergyPlusFixture, ReverseTranslator_OutputTableSummaryReports) {
```

Comment:
I personally prefer the RT test to be in `src/energyplus/Test/OutputTableSummaryReports_GTest.cpp`. I find that easier to find and more logical.